app/views/users/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Create New User</h1>
+
+<%= form_for(@user) do |f| %>
+
+  <%= f.label :email %>
+  <%= f.text_field :email %>
+
+  <%= f.label :password %>
+  <%= f.password_field :password %>
+
```

Comment:
If you were feeling fancy you could add password_confirmation too. Way to go on the f.password_field though.